src/screens/Customers/CustomerMain.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { CustomerRecord } from '../../utils/airtable/interface';
+import { getAllCustomers } from '../../utils/airtable/requests';
+import { Link } from 'react-router-dom';
+
+interface CustomerMainProps {
+  match: any;
+}
+
+function CustomerMain(props: CustomerMainProps) {
```

Comment:
Could we use `RouteComponentProps` here instead? https://stackoverflow.com/questions/48138111/what-typescript-type-should-i-use-to-reference-the-match-object-in-my-props . If not, it's no big deal, but I'd rather just refrain from using `any` if we're going to commit to typescript